responses/__init__.py

```diff
@@ -1,25 +1,24 @@
-from http import client
-
 import inspect
-from http import cookies
 import json as json_module
 import logging
+from collections import namedtuple
+from collections.abc import Sequence
+from collections.abc import Sized
+from functools import wraps
+from http import client
+from http import cookies
```

Comment:
Shouldn't these be grouped?

Reply:
@markstory 
this is better in terms of visibility when you add/remove imports. GIT handles this as separate lines, so you easily see what was added per PR

for human readability it is also fine, since everything is sorted alphanumerically.

Reply:
here is another lib, that I would not use due to some issues that I faced during pipelines
but it has a pretty good explanation: https://github.com/asottile/reorder_python_imports#why-this-style